src/browser/appwindow.js

```diff
@@ -89,6 +90,15 @@ AppWindow.prototype.selectTargetDirectory = function () {
   }.bind(this));
 };
 
+AppWindow.prototype.openProject = function (cwd) {
+  if (!cwd) {
+      return;
+  }
+
+  shell.showItemInFolder(cwd);
+  shell.beep();
```

Comment:
Why playing a sound? Wouldn't do that, because at least for me it's a "something went wrong" sound.